WildShape-Sheets-API/Services/PlayerCharacterService.cs

```diff
@@ -1,42 +1,39 @@
-﻿using Microsoft.Extensions.Options;
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
 using MongoDB.Driver;
 using WildShape_Sheets_API.Models;
 
 namespace WildShape_Sheets_API.Services
 {
     public class PlayerCharacterService
     {
-        private readonly IMongoCollection<PlayerCharacter> playerChars;
+        private readonly DataBaseService _dataBaseService;
+        private readonly IConfiguration _configuration;
 
-        public PlayerCharacterService(IOptions<WildshapeSheetsDBSettings> wildshapeSheetsDBSettings, IConfiguration configuration)
+        public PlayerCharacterService(IConfiguration configuration, DataBaseService dataBaseService)
         {
-            var mongoClient = new MongoClient(
-                wildshapeSheetsDBSettings.Value.ConnectionString);
+            _dataBaseService = dataBaseService;
```

Comment:
Woot! Model services have no knowledge or care about the underlying database implementation. Mongo or Postgresql or Chocolate Chip, it is abstracted away and could be changed to anything in one place (in theory).